src/components/atoms/typography/ms-typography.ts

```diff
@@ -0,0 +1,57 @@
+import { LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+
+import { WithTheme } from '~/mixins'
+
+import { msCheckboxStyles } from './ms-typography.styles'
+
+@customElement('ms-typography')
+export class MSTypography extends WithTheme(LitElement) {
+  static styles = msCheckboxStyles
+
+  @property({ type: String })
```

Comment:
If i'm not wrong, 'id' is already part of LitElements attributes, because of LitElement is an extension of HTMLElement

Reply:
yes, but in this case is needed to have a reference to attach to the new element created, that's why is send it as a property